docs/deploy.md

```
@@ -5,10 +5,36 @@
 docker build -t gachi-be:latest .
 ```
 
-## 2. Run with env file
+## 2. SMTP 비밀번호 주입 방식 선택
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)